CONTRIBUTING.md

```diff
@@ -46,6 +46,34 @@ Your ideas, documentation and code have become an integral part of this project.
 
 In fact, feel free to open a PR to add your name to the [`AUTHORS`](AUTHORS.md) file and get eternal attribution.
 
+## Releasing a new version
+
+1. Review state of the 'develop' branch
+    - Ensure all changes intended for release are merged
+2. Create a release branch
+    - From 'develop', git checkout -b "release-$MAJOR.$MINOR.$PATCH"
+3. Update the new release
+    - [ ] Update version number in `_config.yml` and `README.md`
+    - [ ] Update [`AUTHORS.md`](AUTHORS.md) with new contributors
+    - [ ] Update [`CHANGELOG.md`](CHANGELOG.md)
+    - [ ] Perform extra pass on diff to the 'master' branch
+        - Reread any section or paragraph to ensure wording changes still fit the whole and do not contain grammar or spelling errors
+        - If needed, commit changes and repeat extra pass
+    - [ ] Push branch, open a pull request to the 'master' branch
+        - Request review from multiple reviewers, especialy a proofreader
+    - [ ] Generate new PDF of the primary content
+        - Serve html content with `script/serve.sh`
+        - Optionally, for a visual pre-check, navigate to http://127.0.0.1:4000/ in a browser
+        - Generate the pdf with `script/pdf.sh`
+    - [ ] Create new cover for print
+    - [ ] Create new cover for web PDF
+    - [ ] Create PDF for GitHub release with front and back cover included
+4. Send the files for print to the printer
+    - [ ] Cover file
+    - [ ] Inside pages PDF
+5. Create GitHub release with the release notes and version number
```

Comment:
This point involves creating a Git tag, which should be named the exact version number.

After this the release branch needs to be merged with the `main`.